app/controllers/exam_templates_controller.rb

```diff
@@ -69,6 +69,22 @@ def update
     redirect_to action: 'index'
   end
 
+  def generate
+    copies = params[:numCopies].to_i
+    index = params[:examTemplateIndex].to_i
+    assignment = Assignment.find(params[:assignment_id])
+    assignment_name = assignment.short_identifier
+
+    exam_template = assignment.exam_templates.find(params[:id])
+    exam_template.generate_copies(copies, index)
+    flash_message(:success, t('exam_templates.generate.success', copies: copies))
+
+    generated_filename = "#{index}-#{index + copies - 1}.pdf"
+    send_file("#{EXAM_TEMPLATE_DIR}/#{assignment_name}/#{generated_filename}",
```

Comment:
The `base_path` method is private so I cant access it from the controller, should I make it public?